rtvi-client-android/src/main/java/ai/rtvi/client/helper/LLMHelper.kt

```diff
@@ -61,7 +61,7 @@ class LLMHelper(private val callbacks: Callbacks) : VoiceClientHelper() {
                                 LLMFunctionCallResult(
                                     functionName = data.functionName,
                                     toolCallId = data.toolCallId,
-                                    args = data.args,
+                                    arguments = data.args,
```

Comment:
It seems I will need to push the same fix for iOS 😅:
https://github.com/rtvi-ai/rtvi-client-ios/blob/2df4a5cc2c8ff17ff5a404adf156503a38a74aa5/Sources/RTVIClientIOS/helper/LLMHelper.swift#L31